File: SoftUni Lessons/Python Development/Python Basics April 2019/Past Exams and Problems/9 - 20 and 21 April 2019/02. Easter Party.py
people = int(input())
price_one = float(input())
budget = float(input())

cake = budget * 0.1
price = people * price_one

if 10 <= people <= 15:
    price -= price * 0.15
elif 15 < people <= 20:
    price -= price * 0.20
elif people > 20:
    price -= price * 0.25

total = price + cake

if budget >= total:
    print(f"It is party time! {abs(budget - total):.2f} leva left.")
else:
    print(f"No party! {abs(total - budget):.2f} leva needed.")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/21. МИНАЛИ ИЗПИТИ/01. Online Exam - 20 and 21 April 2019/02. Easter Party.js
function easterParty(params) {
    let countGuests = Number(params.shift());
    let priceOnePerson = Number(params.shift());
    let budget = Number(params.shift());

    if (countGuests >= 10 && countGuests <= 15) {
        priceOnePerson *= 0.85;
    } else if (countGuests > 15 && countGuests <= 20) {
        priceOnePerson *= 0.8;
    } else if (countGuests > 20) {
        priceOnePerson *= 0.75;
    }

    let total = (priceOnePerson * countGuests) + (budget * 0.1);

    if (budget >= total) {
        console.log(`It is party time! ${(budget - total).toFixed(2)} leva left.`);
    } else {
        console.log(`No party! ${(total - budget).toFixed(2)} leva needed.`);
    }
}


easterParty(['18', '30', '450']);  // Expected Output: No party! 27.00 leva needed.
easterParty(['8', '25', '340']);  // Expected Output: It is party time! 106.00 leva left.
easterParty(['24', '35', '550']);  // Expected Output: No party! 135.00 leva needed.